docs/elasticsearch/tasksapis.md

```diff
@@ -0,0 +1,232 @@
+---
+layout: default
+title: Tasks API
+parent: Elasticsearch
+nav_order:  8
+has_math: false
+---
+
+# Tasks API
+
+A task is any operation you execute in a cluster. For example, searching your data for a particular query is a task. When you run Open Distro for Elasticsearch, a task is automatically created to monitor your cluster's health and performance. For more information about all of the tasks currently executing in your cluster, you can use the `tasks` API.
+
+This request returns information about all of your tasks:
+
+```
+GET _tasks
+```
+
+By including a task ID, you can get information specific to a particular task. Note that a task ID consists of a node's identifying string and the task's numerical ID. For example, if your node's identifying string is `nodestring` and the task's numerical ID is `1234`, then your task ID is `nodestring:1234`. You can find this information by running the `_tasks` API.
+
+```
+GET _tasks/<task_id>
+```
+
+Note that if a task finishes executing, it will not be returned as part of your request. For an example of a task that takes a little longer to finish, you can run the [`_reindex`](../reindex-data.md) API on a larger document, and then run the `_tasks` API.
+
+Sample Response:
+```json
+{
```

Comment:
This is a total nitpick, but I like two space indentation on code samples, especially JSON. You can use something like https://jsonlint.com/?code= to handle the formatting and then do a regex find and replace in your text editor, subbing \t for two spaces.